.ado/templates/e2e-testing-uwp.yml

```diff
@@ -17,58 +17,67 @@ steps:
     name: UwpCertificate
     inputs:
       secureFile: FluentTester_TemporaryKey.pfx
-    displayName: "Download UWP App Certificate"
+    displayName: 'Download UWP App Certificate'
 
   - powershell: |
       certutil –f –p password –importpfx $(UwpCertificate.secureFilePath)
-    displayName: "Add Certificate to Personal Store"
+    displayName: 'Add Certificate to Personal Store'
 
   - task: DownloadSecureFile@1
     name: AzureCredentials
     inputs:
-      secureFile: "installcredprovider.ps1"
-    displayName: "Download Azure Credentials Plugin for NuGet"
+      secureFile: 'installcredprovider.ps1'
+    displayName: 'Download Azure Credentials Plugin for NuGet'
 
   - task: PowerShell@2
     inputs:
       filePath: $(AzureCredentials.secureFilePath)
-      arguments: "-AddNetfx -Force"
-    displayName: "Install Azure Credentials Plugin for NuGet"
+      arguments: '-AddNetfx -Force'
+    displayName: 'Install Azure Credentials Plugin for NuGet'
 
   - script: |
       mkdir windows\target
       mkdir windows\target\x64
       mklink /J windows\target\x64\Release windows\x64\Release\FluentTester
     workingDirectory: apps\windows
-    displayName: "Workaround RNW CLI bug"
+    displayName: 'Workaround RNW CLI bug'
 
   - script: |
-      yarn windows --logging --release --arch x64
+      yarn install-windows-test-app
     workingDirectory: apps\windows
-    displayName: "run UWP app"
+    displayName: 'generate UWP app'
 
-  # Kill FluentTester, leave server up and running. This was the only way I could get the server continuously running
+  - script: |
+      yarn windows --logging --arch x64
+    workingDirectory: apps\windows
+    displayName: 'run UWP app'
+
+  - powershell: |
+      Get-Process
+    displayName: 'Get List of processes'
+
+  # Kill ReactTestApp, leave server up and running. This was the only way I could get the server continuously running
```

Comment:
In PowerShell, wouldn't one use something like [`Start-Process`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/start-process?view=powershell-7.1) to fork a new process? Starting up the test app seems like a really expensive workaround…

Reply:
I didn't write this, but I think the interntion was to test that the test app does run, and then do other stuff after

Reply:
The app gets started, then is immediately killed afterwards. There is no time to test anything. The comment also says "This was the only way I could get the server continuously running".

Pinging @samuelfreiberg, since he wrote it.

Reply:
Sorry for such a delayed response, not sure it matters anymore, but I couldn't get the server to run when simply starting it like normal. When using "yarn start" or whatever the command was, it wasn't starting the server. The only way I could get it to run at the time was like this. Running the test app also started the server, but then I had to kill the test app or the E2E tests wouldn't run properly. It was a 'hack' and maybe it's changed now, but it was very strange at the time.